### ISSUE TITLE
Insufficient color contrast

### ISSUE THREAD

**Issue by AmyJuneH** [badcamp/badcamp-2019#5]
https://dev-badcamp-2019.pantheonsite.io/user/login
The color combination #0077a5/#000000 has a contrast ratio of 4.18, which is not sufficient. At this size, you will need a ratio of at least 4.5.
 
Relevant code:
<a href="/user/password">Recover your password</a>

**Comment by sean-e-dietrich** [comment]
Closing as old theme
